content/yubikey-login.md

```diff
@@ -15,98 +15,94 @@ tableOfContents: true
+To do this you must install the yubikey packages, configure a challenge-response slot on the Yubikey, and then configure the necessary PAM modules.
 
-**NOTE:** This guide assumes you are running Pop!_OS
+**NOTE:** Open an additional root terminal: `sudo su`
 
 ## Install Packages
 
 To install the necessary packages, run:
 
-    sudo apt install -y libpam-yubico yubikey-personalization
-
-You may get a question about the PAM configuration line. If so, enter this line:
-
-    mode=challenge-response
-
-Don't enable any PAM modules yet.  We'll do that at the end.
-
-If you have already installed the package or want to reconfigure it, use this command:
-
-    sudo dpkg-reconfigure libpam-yubico
+    sudo apt install -y libpam-yubico yubikey-personalization yubikey-manager
 
 ## Configure Challenge-Response for your Yubikey
 
-To enable challenge-response on your Yubikey, type the following command:
+To enable challenge-response on your Yubikey in slot 2, type the following command:
 
-    ykpersonalize -2 -ochal-resp -ochal-hmac -ohmac-lt64 -oserial-api-visible
+    ykman otp chalresp -g 2
 
 This configures slot 2 for challenge-response, and leaves slot 1 alone.
 
-Next we need to create some challenge response files, move them to a system wide directory, and secure those files:
+Next we need to create a place to sore your challenge response files, secure those files, and finally create the stored challenge files:
 
     sudo mkdir /var/yubico
-    sudo chown root.root /var/yubico
+    sudo chown root /var/yubico
     sudo chmod 700 /var/yubico
     ykpamcfg -2 -v
 
 You should receive a message similar to:
 
 `Stored initial challenge and expected response in '$HOME/.yubico/challenge-123456'.`
 
-You should receive a unique *challenge-123456* in your output.
+You should receive a unique *challenge-serial* in your output.
 
 Now, to finish up:
 
     sudo mv ~/.yubico/challenge-123456 /var/yubico/alice-123456
-    sudo chown root.root /var/yubico/alice-123456
-    sudo chmod 600 /var/yubico/alice-123456
+    sudo chown root.root /var/yubico/bob-123456
+    sudo chmod 600 /var/yubico/lisa-123456
 
-Pay close attention when copying/pasting the commands above.  The *challenge-123456* and *alice-123456* needs to match whatever your output is.**
+Pay close attention when copying/pasting the commands above.  The *challenge-123456* and *alice-123456* needs to match the both the output from the `ykpamcfg` command and the final file needs to match the name of your user name and serial.**
 
 ## Configure Plugable Authentication Modules
 
 **Before making any changes to the files listed below, I highly recommend backing up each file, and having a sudo/root session open in case you need to roll-back.**
 
-You need to add the following line to each of the files listed below:
+    sudo dpkg-reconfigure libpam-yubico
+
+You will want to change the "Parameters for Yubico PAM:" to be:
+
+    mode=challenge-response debug chalresp_path=/var/yubico
```

Comment:
Ah, I see, this is covered later. I wonder if a note about the `debug` setting, with a link to the "debugging and testing" section, would be appropriate here, just in case readers have a short attention span like I do.